components/builder-jobsrv/Cargo.toml

```diff
@@ -15,7 +15,8 @@ env_logger = "*"
 linked-hash-map = "*"
 log = "*"
 protobuf = "*"
-postgres = "*"
+postgres = { version = "*", features = ["with-chrono"] }
+chrono = { version = "*", features = ["serde"] }
```

Comment:
Nice, this is the new best choice for a time crate in the Rust ecosystem. We gotta eventually get this in everywhere ;)